ts/src/okx.ts

```diff
@@ -2907,9 +2907,9 @@ export default class okx extends Exchange {
             const marketId = this.safeString (rawOrder, 'symbol');
             const type = this.safeString (rawOrder, 'type');
             const side = this.safeString (rawOrder, 'side');
-            const amount = this.safeValue (rawOrder, 'amount');
-            const price = this.safeValue (rawOrder, 'price');
-            const orderParams = this.safeValue (rawOrder, 'params', {});
+            const amount = this.safeFloat (rawOrder, 'amount');
+            const price = this.safeFloat (rawOrder, 'price');
```

Comment:
@rayBastard this is complicated, I would leave `safeValue` here for now  (it's an exception)

Reply:
Agree, returned safeValue